velox/type/tests/DecimalTest.cpp

```diff
@@ -68,5 +68,12 @@ TEST(DecimalTest, decimalToString) {
       DecimalUtil::toString(UnscaledLongDecimal(-1000), DECIMAL(20, 6)));
   ASSERT_EQ("0", DecimalUtil::toString(UnscaledLongDecimal(0), DECIMAL(20, 9)));
 }
+
+TEST(DecimalTest, overloads) {
+  ASSERT_EQ(UnscaledShortDecimal(3), UnscaledShortDecimal(10) / 3);
+  ASSERT_EQ(UnscaledLongDecimal(33), UnscaledLongDecimal(100) / 3);
+  ASSERT_EQ(UnscaledLongDecimal(300), UnscaledLongDecimal(100) * 3);
```

Comment:
* operator doesn't work in fb internal tests. We can disable this and I can help on turn on this later.

Reply:
Can I use the mul https://github.com/facebookincubator/velox/blob/main/velox/type/DecimalUtil.h#L33 ?

Reply:
I also want to discuss this issue further here https://github.com/facebookincubator/velox/issues/2388